app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    text = command.split()
```

Comment:
Check for the length of the value obtained after the `split` and return unpacking that was present in previous commits. Besides that, use values obtained in the unpacking in the if condition on line 3 and in the context manager to avoid indexing.